vic5_manuscript.tex

```diff
@@ -193,6 +193,10 @@ \section{VIC Infrastructure}
       \item The \textit{Python Driver} provides Python bindings to the functions and data structures of VIC’s physical core. These functions are made available as Python functions using the C Foreign Function Interface for Python (CFFI; \url{http://cffi.readthedocs.io}). Although the \textit{Python Driver} is currently only used for testing, this driver is uniquely suited for future interactive classroom sessions, data assimilation, or large ensemble applications.
     \end{enumerate}
 
+    Additionally, VIC includes a set of general "Shared Driver Utilities" that may be used by the individual drivers. These utilities are ...
+
+    % TODO - finish that thought...

```

Comment:
Yes - please finish so we can do a final review.

Reply:
done.